examples/express-ts/test.js

```diff
@@ -0,0 +1,357 @@
+const axios = require('axios');
```

Comment:
🥇 thanks for writing this! 🎉

Reply:
You're welcome! Thanks for writing `express-ts` and the `.sh` I based this on!